spec/features/user/logged_in_user_sees_git_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+describe 'A registered user' do
+  context 'when I visit /dashboard' do
+    it 'sees a section for github' do
+      VCR.use_cassette("all the things") do
```

Comment:
I wonder if there's a better use of VCR? it seems a bit repetitious the way we're using it.